tensorflow_serving/sources/storage_path/file_system_storage_path_source.cc

```diff
@@ -76,7 +76,83 @@ std::set<string> GetDeletedServables(
   return deleted_servables;
 }
 
-// Like PollFileSystemForConfig(), but for a single servable.
+//Add a new ServableData for the model version to the the vector of versions to aspired
+void UpdateVersionToAspire(
+    const FileSystemStoragePathSourceConfig::ServableToMonitor& servable,
+    const string version_relative_path,
+    const int version_number,
+    std::vector<ServableData<StoragePath>>* versions) {
+    const ServableId servable_id = {servable.servable_name(), version_number};
+    const string full_path =
+        io::JoinPath(servable.base_path(), version_relative_path);
+    versions->emplace_back(ServableData<StoragePath>(servable_id, full_path));
+}
+
+bool IsValidVersionNumber(const string& version_path, int64 *version_number) {
```

Comment:
Nice!

1. Please rename to ParseVersionNumber.
2. Add documentation along the lines of: attempts to parse ..., upon success writes the result to 'version_number' and returns true; else returns false